src/app/views/authentication/profile/Profile.tsx

```diff
@@ -103,11 +104,17 @@ export class Profile extends Component<IProfileProps, IProfileState> {
       graphExplorerMode,
     }: any = this.props;
 
+    const isDelegatedPermissions = this.props.permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
+    const userText = user.displayName + ' '
+      + (isDelegatedPermissions
+        ? translateMessage('As user')
+        : translateMessage('As Teams app'));
+
```

Comment:
@lhao03 I was looking at your screen recording and I don't see anywhere that uses the "As Teams app" text, is this still in use?

Reply:
Nope not is use, was blindly copying @_@

Reply:
Do you think we need the (As user at all anymore?)